rslaser/pulse/pulse.py

```diff
@@ -180,76 +188,112 @@ class LaserPulseSlice(ValidatorBase):
+        # GsnBm.x = 0 #Transverse Positions of Gaussian Beam Center at Waist [m]
+        # GsnBm.y = 0
+        
+        # GsnBm.z = params.slice_params.propLen + self._pulse_pos #Longitudinal Position of Waist [m]
+        # GsnBm.xp = 0 #Average Angles of Gaussian Beam at Waist [rad]
+        # GsnBm.yp = 0
+        # GsnBm.avgPhotEn = self.phE #Photon Energy [eV]
+        # GsnBm.pulseEn = params.slice_params.pulseE*np.exp(-self._pulse_pos**2/(2*sig_s**2)) #Energy per Pulse [J] - to be corrected
+        # GsnBm.repRate = 1 #Rep. Rate [Hz] - to be corrected
+        # GsnBm.polar = params.slice_params.poltype #1- linear horizontal?
+        # GsnBm.sigX = params.slice_params.sigrW #Horiz. RMS size at Waist [m]
+        # GsnBm.sigY = GsnBm.sigX #Vert. RMS size at Waist [m]
+
+        # GsnBm.sigT = 10e-15 #Pulse duration [s] (not used?)
+        # GsnBm.mx = params.slice_params.mx #Transverse Gauss-Hermite Mode Orders
+        # GsnBm.my = params.slice_params.my
 
         #***********Initial Wavefront
-        _wfr = srwlib.SRWLWfr() #Initial Electric Field Wavefront
-        _wfr.allocate(1, 1000, 1000) #Numbers of points vs Photon Energy (1), Horizontal and Vertical Positions (dummy)
-        _wfr.mesh.zStart = 0.0 #Longitudinal Position [m] at which initial Electric Field has to be calculated, i.e. the position of the first optical element
-        _wfr.mesh.eStart = GsnBm.avgPhotEn #Initial Photon Energy [eV]
-        _wfr.mesh.eFin = GsnBm.avgPhotEn #Final Photon Energy [eV]
-
-        _wfr.unitElFld = 1 #Electric field units: 0- arbitrary, 1- sqrt(Phot/s/0.1%bw/mm^2), 2- sqrt(J/eV/mm^2) or sqrt(W/mm^2), depending on representation (freq. or time)
-
-        distSrc = _wfr.mesh.zStart - GsnBm.z
-        #Horizontal and Vertical Position Range for the Initial Wavefront calculation
-        #can be used to simulate the First Aperture (of M1)
-        #firstHorAp = 8.*rmsAngDiv*distSrc #[m]
-        xAp = 8.*sigrL
-        yAp = xAp #[m]
-
-        _wfr.mesh.xStart = -0.5*xAp #Initial Horizontal Position [m]
-        _wfr.mesh.xFin = 0.5*xAp #Final Horizontal Position [m]
-        _wfr.mesh.yStart = -0.5*yAp #Initial Vertical Position [m]
-        _wfr.mesh.yFin = 0.5*yAp #Final Vertical Position [m]
-
-        sampFactNxNyForProp = params.slice_params.sampFact #sampling factor for adjusting nx, ny (effective if > 0)
-        arPrecPar = [sampFactNxNyForProp]
-
-        srwlib.srwl.CalcElecFieldGaussian(_wfr, GsnBm, arPrecPar)
-
-        ##Beamline to propagate to waist ( only if d_to_w(t=0) < d_to_w_cutoff )
-        if params.d_to_w < d_to_w_cutoff:
-          optDriftW=srwlib.SRWLOptD(params.slice_params.propLen)
-          propagParDrift = [0, 0, 1., 0, 0, 1.1, 1.2, 1.1, 1.2, 0, 0, 0]
-          optBLW = srwlib.SRWLOptC([optDriftW],[propagParDrift])
-          srwlib.srwl.PropagElecField(_wfr, optBLW)
-        self.wfr = _wfr
+        # _wfr = srwlib.SRWLWfr() #Initial Electric Field Wavefront
```

Comment:
Unless a commented out line of code is crucial to keep, they should be removed from the module